wallet/how-to/get-started-building/run-devnet.md

```diff
@@ -31,20 +47,16 @@ You can manage multiple seed phrases by using multiple browser profiles, each wi
 MetaMask installation.
 :::
 
-In the **Server** setting of your workspace, find the hostname and port of your Ganache
-network, which comprises the RPC URL of your network:
-
-```text
-http://<hostname>:<port>
-```
+To run a Hardhat development network and expose a JSON-RPC interface, run `npx hardhat node`.
+You can now connect MetaMask to your development network RPC URL, `http://127.0.0.1:8545/`.
 
-In the MetaMask extension, connect to your Ganache network:
+In the MetaMask extension:
 
-1. Select the network you're currently connected to.
+1. In the upper left corner, select the network you're currently connected to.
 1. Select **Add network**.
 1. Select **Add a network manually**.
-1. Enter the RPC URL of your network.
-1. Enter MetaMask's default [chain ID](../connect/detect-network.md#chain-ids) for Ganache, `1337`.
+1. Enter the RPC URL of your Hardhat network, `http://127.0.0.1:8545/`.
+1. Enter MetaMask's default chain ID for Hardhat, `31337`.
```

Comment:
https://hardhat.org/hardhat-network/docs/metamask-issue
https://github.com/MetaMask/metamask-extension/issues/10290

just want to make sure our instructions for adding Hardhat to MetaMask are correct